roles/girder-variables/tasks/main.yml

```diff
@@ -13,6 +13,11 @@
             {{ state == "stopped" or state == "started" or
                state == "restarted" or state == "reloaded" }}
 
+  - name: girder | venv root | default | set
+    set_fact:
+      girder_venv_root: /opt/girder/libs
+    when: girder_venv_root == ""
```

Comment:
Having the virtualenv under girder is fine for now, but I'd rather have it as an installed "package" that sits alongside all the other packages, where instead of using
`/opt/{{ package_name }}/{{ package_version }}`, we can use `/opt/virtual-env/{{ env_name }}` and have a variable in roles like `girder` to point to them.